### ISSUE TITLE
invalid output when parsing fields with fieldNo > 100

### ISSUE THREAD

**Issue by brokenmass** [lal12/proto-parse#3]
the following message

```protobuf
message Allocation {
  Test test = 9876;
  string start_date = 18;
  string end_date = 19;
}
```

generates this output
```js
{ type: 'message',
  name: 'Allocation',
  content:
  [ { type: 'field',
      repeated: false,
      typename: 'Test',
      name: 'test',
      fieldNo: 99,
      opts: {} },
    { type: 'field',
      repeated: false,
      typename: 'string',
      name: 'start_date',
      fieldNo: 18,
      opts: {} },
    { type: 'field',
      repeated: false,
      typename: 'string',
      name: 'end_date',
      fieldNo: 19,
      opts: {} } ],
  opts: {} }
```

as you can see `fieldNo` key is `98` instead of `9876`.

**Comment by brokenmass** [comment]
fix:

```js
decimalLit = a:("0" / [1-9] decDigit*)
{ return parseInt(a[0]+ (a[1] ? a[1].join("") : "")) }
```

**Comment by lal12** [comment]
Sorry didn't saw that you already provided the fix. I hadn't time 'til now to investigate it.

Fixed in 0.0.5
